api_client/python/timesketch_api_client/search.py

```diff
@@ -573,6 +574,28 @@ def description(self, description):
         self._description = description
         self.commit()
 
+    @property
+    def expected_size(self):
```

Comment:
Why do we need this in the first place?

Reply:
explained offline, for automation in order to be able to effectively set the max_entries size